frontend/assets/css/style.css

```diff
@@ -130,10 +130,18 @@ body {
     */
 }
 
+div .background {
+    -webkit-box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.75);
+    -moz-box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.75);
+    box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.75);
```

Comment:
the shadow is too much `0.75` try `0.4`, then 
3px`(x axis)` 4px`(y axis)` 5px`(reduce the blur)` -2px`(reduce the shadow spreading)`

to make the shadow, be on bottom and right based on `x and y axis`